### ISSUE TITLE
Zenodo response problem, finding filenames in file list

### ISSUE THREAD

**Issue by zephyris** [zephyris/tryptag#19]
Zenodo appears to have changed the response listing files, "filename" no longer being an attribute of files in a Zenodo response. Error   below from [https://github.com/zephyris/tryptag/blob/main/examples/localisation_search.ipynb]

```
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
[<ipython-input-3-c6debed5e87f>](https://localhost:8080/#) in <cell line: 9>()
      7 print("Query:", query)
      8 print("")
----> 9 hits = tryptag.localisation_search(query)
     10 
     11 # result is a list of hits, in the form {"gene_id": gene_id, "terminus": terminus}

4 frames
[/usr/local/lib/python3.10/dist-packages/tryptag/tryptag.py](https://localhost:8080/#) in localisation_search(self, query_term, life_stage, match_subterms, exclude_modifiers, required_modifiers)
    510     # check all against query
    511     hits = []
--> 512     for cell_line in self.worklist_all(life_stage):
    513       if self._localisation_match(cell_line,  query_term, match_subterms=match_subterms, exclude_modifiers=exclude_modifiers, required_modifiers=required_modifiers):
    514         hits.append(cell_line)

[/usr/local/lib/python3.10/dist-packages/tryptag/tryptag.py](https://localhost:8080/#) in worklist_all(self, life_stage)
    380         terminus=terminus,
    381        ) for gene_id, terminus in
--> 382       ((gene_id, terminus) for gene_id, gene_entry in self.gene_list[life_stage].items() for terminus in self.termini if terminus in gene_entry)
    383     ]
    384 

[/usr/lib/python3.10/functools.py](https://localhost:8080/#) in __get__(self, instance, owner)
    979                 val = cache.get(self.attrname, _NOT_FOUND)
    980                 if val is _NOT_FOUND:
--> 981                     val = self.func(instance)
    982                     try:
    983                         cache[self.attrname] = val

[/usr/local/lib/python3.10/dist-packages/tryptag/tryptag.py](https://localhost:8080/#) in gene_list(self)
    331     # download localisations.tsv from master record
    332     zenodo_json = self._fetch_zenodo_record_json(self.zenodo_record_id)
--> 333     lines = self._fetch_zenodo_record_file(zenodo_json, "localisations.tsv").splitlines()
    334     # load into default life stage
    335     # TODO: Handle loading of multiple different life stages

[/usr/local/lib/python3.10/dist-packages/tryptag/tryptag.py](https://localhost:8080/#) in _fetch_zenodo_record_file(self, zenodo_json, file_name)
    263     from urllib.error import HTTPError
    264     for file in zenodo_json["files"]:
--> 265       if file["filename"] == file_name:
    266         url = f"[https://zenodo.org/api/records/{zenodo_json['record_id']}/files/{file_name}/content](https://zenodo.org/api/records/%7Bzenodo_json['record_id']%7D/files/%7Bfile_name%7D/content)"
    267         if self.print_status: print("  Fetching file "+file_name+" from: "+url)

KeyError: 'filename'
```

**Comment by zephyris** [comment]
@ulido

**Comment by ulido** [comment]
Ok I'll take care of it.

**Comment by zephyris** [comment]
Fixed in 651e391
